web/static/manifest.json

```diff
@@ -0,0 +1,579 @@
+{
+  "name": "JioTV Go",
+  "theme_color": "#000000",
+  "background_color": "#000000",
```

Comment:
We need to to give more thought here.

How will it work for light and dark mode in our case?